fixed-price-sale/program/Cargo.toml

```diff
@@ -29,3 +30,4 @@ solana-program = "~1.9.28"
 solana-sdk = "~1.9.28"
 spl-associated-token-account = "~1.0.3"
 mpl-testing-utils= {path="../../core/rust/testing-utils" }
+mpl-metaplex = "0.0.3"
```

Comment:
Why is this needed? This module is super outdated .. I recently removed it from auction house.

Reply:
`mpl-metaplex` is used by some Fixed Price Sale tests.

Reply:
OK, actually I found the same to be true in auction house, but there it was using a struct that we now expose via mpl-token-metadata. Maybe something like this is possible here as well?

Reply:
Rewriting Fixed Price Sale or a large refactor of the tests is out-of-scope for this PR and would be better in a separate one. This is the smallest change necessary to get these tests to pass.

Reply:
Ah ok in that case we can just leave it in. I thought it was a simple `use` statement that'd need to be replaced. If it would be more tricky I agree that it should be in a separate PR.